depthmapX/treeWindow.cpp

```diff
@@ -48,16 +46,16 @@ indexWidget::indexWidget(QWidget *parent, bool custom)
         SLOT(OnSelchangingTree(QTreeWidgetItem*, int)));
 }
 
-indexWidget::~indexWidget()
+IndexWidget::~IndexWidget()

```

Comment:
Do we need an empty non-virtual destructor? What happens if you just remove it, the compiler should provide an equivalent default one...